services/blockchain-indexer/shared/indexer/indexStatus.js

```diff
@@ -46,7 +46,8 @@ const getIndexStats = async () => {
 		const currentChainHeight = await getCurrentHeight();
 		const genesisHeight = await getGenesisHeight();
 		const numBlocksIndexed = await blocksDB.count();
-		const [lastIndexedBlock] = await blocksDB.find({ sort: 'height:desc', limit: 1 }, ['height']);
+		// TODO: Revert change once microservice dependency feature is ready
+		const [lastIndexedBlock = {}] = await blocksDB.find({ sort: 'height:desc', limit: 1 }, ['height']);
```

Comment:
Since the dependency feature is already in place, can we please address the TODO?